### ISSUE TITLE
prevent duplicate object creation from double clicks

### ISSUE THREAD

**Issue by nbeernink** [jstanden/cerb#119]
Quickly double clicking on "ok" in the schedule behavior causes the task to run twice even though the intend is to run it only once.

**Comment by jstanden** [comment]
This is related to another issue where we should show a spinner for long-running behaviors.  At the same time we should disable the OK button.

**Comment by svenamaysim** [comment]
This is the same for user creation - resulted a few times in two users being created on our side

**Comment by jstanden** [comment]
The scheduled behavior dialog no longer exists in 7.4 since it's handled by conversational bots.

But for record creation, we can definitely do something to block double submits.

**Comment by jstanden** [comment]
This was implemented in 9.2. On all editor popups, clicking 'Save changes' displays a spinner and disables the button.
